### PR TITLE
relogin fixes

### PR DESCRIPTION
**Pull request by chrisnojima**
@keybase/react-hackers this fixes some issues that the switch users command reveals
`keybase login -s other_account` to test

### CODE REVIEW

File: shared/actions/config/index.js [hunk 1]
```diff
@@ -476,16 +476,10 @@ function* configSaga(): Saga.SagaGenerator<any, any> {
...
   yield* Saga.chainGenerator<
-    | ConfigGen.LoggedInPayload
-    | ConfigGen.LoggedOutPayload
-    | ConfigGen.DaemonHandshakePayload
-    | GregorGen.UpdateReachablePayload
-  >(
-    [ConfigGen.loggedIn, ConfigGen.loggedOut, ConfigGen.daemonHandshake, GregorGen.updateReachable],
-    loadDaemonBootstrapStatus
-  )
+    ConfigGen.LoggedInPayload | ConfigGen.DaemonHandshakePayload | GregorGen.UpdateReachablePayload
```
Comment (chrisnojima) [inline shared/actions/config/index.js]: on logout this just loads all blank data which isn't useful. its already cleared from the store

File: shared/actions/json/config.json [hunk 2]
```diff
@@ -99,7 +99,10 @@
     "persistRoute": {"path": "Array<any>"},
     "setNavigator": {"navigator": "any"},
     "loggedOut": {},
-    "loggedIn": {"causedByStartup": "boolean"},
+    "loggedIn": {
+      "description": "Used internally to know we were logged in. if you want to react to being logged in likely you want bootstrapStatusLoaded",
```
Comment (buoyad) [inline shared/actions/json/config.json]: `_description` and `build-actions`

**Comment by chrisnojima** [comment]
@buoyad updated

**Comment by chrisnojima** [comment]
@buoyad bump

**Pull request merged**